### PR TITLE
RavenDB-4836 Add default analyzer as additional parameter to MoreLikeThisQuery

### PR DESCRIPTION
**Pull request by ricardobrandao**

### CODE REVIEW

File: Raven.Tests.Issues/RavenDB_4836.cs [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+namespace Raven.Tests.Issues
+{
+    public class RavenDB_4836 : RavenTestBase
+    {
+        private readonly EmbeddableDocumentStore _documentStore;
+
+        public RavenDB_4836()
+        {
+            _documentStore = NewDocumentStore();
```
Comment (fitzchak) [inline Raven.Tests.Issues/RavenDB_4836.cs]: _documentStore should be disposed. Other than that it looks good.

Comment (ricardobrandao) [inline Raven.Tests.Issues/RavenDB_4836.cs]: @fitzchak done. Also fixed the dispose for test RavenDB_4461. Thanks.

Comment (ricardobrandao) [inline Raven.Tests.Issues/RavenDB_4836.cs]: Just for the record: RavenTestBase [disposes](https://github.com/ayende/ravendb/blob/v3.5/Raven.Tests.Helpers/RavenTestBase.cs#L911) all document stores created using [NewDocumentStore()](https://github.com/ayende/ravendb/blob/v3.5/Raven.Tests.Helpers/RavenTestBase.cs#L270).

Comment (fitzchak) [inline Raven.Tests.Issues/RavenDB_4836.cs]: Yes, this is right. 
But it safer this way, and make it easy to port to future versions.

**Pull request merged**